core/arch/arm/kernel/kern.ld.S

```diff
@@ -116,6 +116,18 @@ SECTIONS
 		__exidx_end = .;
 	}
 
+	.ARM.extab : {
+		__extab_start = .;
+		*(.ARM.extab*)
+		__extab_end = .;
+	}
+
+	.eh_frame : {
+		__eh_frame_start = .;
+		*(.eh_frame*)
+		__eh_frame_end = .;
+	}
+
```

Comment:
These symbols do not appear to be used directly in this patch, why is it needed?
Also, `plat-sunxi/kern.ld.S` is not updated.